### PR TITLE
task 6

### PR DESCRIPTION
**Pull request by Bokbulbok7**

### CODE REVIEW

File: __tests__/integration.test.js [hunk 1]
```diff
@@ -119,3 +119,43 @@ describe("GET/api/articles", () => {
...
+      .get("/api/articles/1/comments")
+      .expect(200)
+      .then(({ body }) => {
+        const { comments } = body;
+        expect(comments).toHaveLength(11);
+        comments.forEach((comment) => {
+          expect(comment).toMatchObject({
+            comment_id: expect.any(Number),
+            author: expect.any(String),
+            article_id: expect.any(Number),
```
Comment (Hard-Luck) [inline __tests__/integration.test.js]: You know the article id

File: __tests__/integration.test.js [hunk 2]
```diff
@@ -119,3 +119,43 @@ describe("GET/api/articles", () => {
...
+
+  it("GET:400 responds with an appropriate error message when given an invalid id", () => {
+    return request(app)
+      .get("/api/articles/not-an-id/comments")
+      .expect(400)
+      .then((response) => {
+        expect(response.body.msg).toBe("Bad request.");
+      });
+  });
+});
```
Comment (Hard-Luck) [inline __tests__/integration.test.js:172]: Need to check comments are sorted by date

**Review by Hard-Luck — CHANGES_REQUESTED** [review]
Needs another test, them you can merge

File: __tests__/integration.test.js [hunk 3]
```diff
@@ -129,10 +129,11 @@ describe("/api/articles/:articleId/comments", () => {
         const { comments } = body;
         expect(comments).toHaveLength(11);
         comments.forEach((comment) => {
+          console.log(comment);
```
Comment (popatre) [inline __tests__/integration.test.js]: remove console log

File: __tests__/integration.test.js [hunk 4]
```diff
@@ -141,6 +142,17 @@ describe("/api/articles/:articleId/comments", () => {
...
 
+  it("GET: 200 response with an array of all comments sorted by date in descending order as default sort values.", () => {
+    return request(app)
+      .get("/api/articles/1/comments")
+      .expect(200)
+      .then(({ body }) => {
+        const { comments } = body;
+        expect(comments).toHaveLength(11);
+        expect(comments).toBeSortedBy("created_at", { descending: true });
+      });
```
Comment (popatre) [inline __tests__/integration.test.js:152]: nice jest sorted test 👍

**Review by popatre — APPROVED** [review]
Looks good, just remove that console log 👍

**Pull request merged**